install/kubernetes/cilium/values.yaml

```diff
@@ -437,12 +437,15 @@ hubble:
   # Enable Hubble (true by default).
   enabled: true
 
-  # Buffer size of the channel Hubble uses to receive monitor events. If this value is not set,
-  # the queue size is set to the default monitor queue size.
+  # Buffer size of the channel Hubble uses to receive monitor events. If this
+  # value is not set, the queue size is set to the default monitor queue size.
   # eventQueueSize: ""
 
-  # Number of recent flows for Hubble to cache. Defaults to 4096.
-  # flowBufferSize: ""
+  # Number of recent flows for Hubble to cache. Defaults to 4095.
+  # Possible values are:
+  #   1, 3, 7, 15, 31, 63, 127, 255, 511, 1023,
+  #   2047, 4095, 8191, 16383, 32767, 65535
+  # eventBufferCapacity: ""
```

Comment:
nit: `eventBufferCapacity` is an integer so shouldn't have example value `""`. I would use the default `4095`.

Reply:
The actual value is anyway quoted when set in cilium's `configMap`. The main reason for this is that large integer values get converted to scientific notation which would turn into an invalid option value. For more information about this, please read [Make Types Clear](https://helm.sh/docs/chart_best_practices/values/#make-types-clear) in Helm's best practices documentation. Note that I did check that `65535` does not get converted to scientific notation but it doesn't hurt using a string in this case to avoid this problem altogether.